tests/console/rmt/rmt_proxy.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: yast2-rmt rmt-server
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test RMT basic function behind proxy.
+# Maintainer: Lemon Li <leli@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use repo_tools;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    record_info('RMT server setup', 'Start to setup a rmt server');
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+
+    # Set RMT behind a proxy (with NTLM Auth), and ensure RMT basic function works.
+    record_info('Set proxy', 'Set RMT behind a proxy (with NTLM Auth)');
+    # set the proxy setting to rmt.conf
+    my $proxy_url = 'http:\/\/' . get_required_var('PROXY_SERVER');
+    my $rmt_conf = '/etc/rmt.conf';
+    my $rmt_conf_bk = '/etc/rmt.conf.bk';
+
+    # backup the rmt conf file
+    assert_script_run("cp $rmt_conf $rmt_conf_bk");
+
+    # update the rmt conf file with the proxy setting
+    assert_script_run("sed -i 's/proxy: /proxy: $proxy_url/g' $rmt_conf");
+    assert_script_run("sed -i 's/proxy_auth: /proxy_auth: ntlm/g' $rmt_conf");
```

Comment:
what's the user or domain for ntlm auth?

Reply:
I suspect this ntlm not used at all, seems tinyproxy do not support ntlm auth.

Reply:
I need to investigate how to setup the NTLM server, so hang the PR for now.

Reply:
@lemon-suse 
Please help add my request into your considerration:)

[SUSE-JIRA] (SLE-22181) QA: Support NTLM authentication in the Unified Installer

Reply:
Sorry, since this PR only set on SUT while not the proxy server, so I think it won't help you for the Jira feature.